### ISSUE TITLE
Different transparency between the top panel and the dock in Cosmic

### ISSUE THREAD

**Issue by eskander** [ubuntu/yaru#890]
Since upgrading to Cosmic beta I noticed the difference in transparency between the top panel and the dock. This didn't happen with the snap so I think it's a bug.

![screenshot from 2018-10-04 23-39-24](https://user-images.githubusercontent.com/11521400/46507117-d7d65980-c82e-11e8-97b2-4394d4f564ab.png)

**Comment by Feichtmeier** [comment]
I don't really know if the upgrade is already supported .

**Comment by eskander** [comment]
Even if it's not supported, one way to help as a community member and an enthusiast is to install the beta and report for any encountered bugs before the final release. At least that's what I think I'm doing.

**Comment by clobrano** [comment]
I observed this some months ago testing the new dock version that was intended for cosmic only in fact (it's the version that should support bright borders on maximized windows). Thanks for reporting this.

**Comment by didrocks** [comment]
That's correct, the dock has a different transparency than top panel in the new dock. @clobrano do you have time to have a look if this could be adjusted via the css override file?

**Comment by clobrano** [comment]
Yes, the fix should be easy. Is this dock version to be shipped on 18.04 as well? Otherwise the fix is for cosmic only
